examples/apexlog/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	"github.com/airbrake/gobrake/v5/apexlog"
+	"github.com/apex/log"
+)
+
+func main() {
+	// Initialize the airbrake notifier
+	airbrake := gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
+		ProjectId:   105138,                             // use the ProjectId of your project
+		ProjectKey:  "fd04e13d806a90f96614ad8e529b2822", // use the ProjectKey of your project
+		Environment: "production",
+	})
+
+	// Close the airbrake notifier
+	defer airbrake.Close()
+
+	// Create the airbrake handler for "github.com/apex/log"
+	// Insert the airbrake notifier instance and set the severity level.
+	// The acceptable severity levels are "DebugLog" "InfoLog", "WarnLog", "ErrorLog", and "FatalLog".
+	// If the severity level is not recognized, "error" severity is used.
+	// Note: The severity is different from level used by log package to make sure that you can send logs to airbrake accordingly.
+	airbrakeHandler := apexlog.New(airbrake, apexlog.ErrorLog)
+
+	log.SetLevel(log.DebugLevel)
+
+	// Set the airbrake handler in the log
+	log.SetHandler(airbrakeHandler)
+
+	ctx := log.WithFields(log.Fields{
+		"file": "something.png",
+		"type": "image/png",
+		"user": "tobi",
+	})
+
+	fmt.Println("Check your Airbrake dashboard at https://airbrake.io/projects/:ProjectId to see these log messages")
```

Comment:
* We should print the real project id being used so that users can actually click on the link
* It would be nice to explain that some of the log messages won't be delivered to Airbrake due to severity settings. As of now this message implies all of the messages will be delivered, which isn't actually the case